Marlin/src/inc/Changes.h

```diff
@@ -579,6 +579,18 @@
   #error "Y_DUAL_STEPPER_DRIVERS is no longer needed and should be removed."
 #elif defined(NUM_Z_STEPPER_DRIVERS)
   #error "NUM_Z_STEPPER_DRIVERS is no longer needed and should be removed."
+#elif defined(SNMM)
+  #error "SNMM is obsolete. Define MMU_MODEL as PRUSA_MMU1 instead."
+#elif defined(MK2_MULTIPLEXER)
+  #error "MK2_MULTIPLEXER is obsolete. Define MMU_MODEL as PRUSA_MMU1 instead."
+#elif defined(PRUSA_MMU2)
```

Comment:
Shouldn't be #elif defined(MMU_MODEL)

Reply:
There was an issue as it happens. It is fixed with 1ee0542357.